samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -168,6 +169,7 @@ static int init(void)
 
 static void reset(void)
 {
+	int err;
```

Comment:
Code guidelines require empty lines after variable declarations
```suggestion
	int err;
	
```

Reply:
Huh, this isn't failing CI. @jhedberg did we change that, or is the CI not running properly?

Reply:
I added the empty line regardless of CI not failing